src/elements/content-preview/PreviewLoading.scss

```diff
@@ -5,10 +5,10 @@
         color: $twos;
         position: relative;
         text-align: center;
-        z-index: 1;
```

Comment:
Not sure if this was doing anything. Caused a bug where it came over the navigation arrows.